From 5f641a86cacb512f8974416773fc6a2e39798f94 Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Tue, 11 Aug 2020 15:39:11 +0700
Subject: [PATCH 500/504] ARM: dts: r8a7742: Correct fallback compat string for
 GPIO

The RZ/G1H SoC (R8A7742) has the same GPIO hardware as RCar Gen2. Thus,
use common rcar-gen2-gpio compat string for it instead of gpio-rcar
which is for RCar Gen1 SoC.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742.dtsi | 18 ++++++++++++------
 1 file changed, 12 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index 91fd952..87952ba 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -284,7 +284,8 @@
 	};
 
 	gpio0: gpio@e6050000 {
-		compatible = "renesas,gpio-r8a7742", "renesas,gpio-rcar";
+		compatible = "renesas,gpio-r8a7742",
+				"renesas,rcar-gen2-gpio";
 		reg = <0 0xe6050000 0 0x50>;
 		interrupts = <0 4 IRQ_TYPE_LEVEL_HIGH>;
 		#gpio-cells = <2>;
@@ -297,7 +298,8 @@
 	};
 
 	gpio1: gpio@e6051000 {
-		compatible = "renesas,gpio-r8a7742", "renesas,gpio-rcar";
+		compatible = "renesas,gpio-r8a7742",
+				"renesas,rcar-gen2-gpio";
 		reg = <0 0xe6051000 0 0x50>;
 		interrupts = <0 5 IRQ_TYPE_LEVEL_HIGH>;
 		#gpio-cells = <2>;
@@ -310,7 +312,8 @@
 	};
 
 	gpio2: gpio@e6052000 {
-		compatible = "renesas,gpio-r8a7742", "renesas,gpio-rcar";
+		compatible = "renesas,gpio-r8a7742",
+				"renesas,rcar-gen2-gpio";
 		reg = <0 0xe6052000 0 0x50>;
 		interrupts = <0 6 IRQ_TYPE_LEVEL_HIGH>;
 		#gpio-cells = <2>;
@@ -323,7 +326,8 @@
 	};
 
 	gpio3: gpio@e6053000 {
-		compatible = "renesas,gpio-r8a7742", "renesas,gpio-rcar";
+		compatible = "renesas,gpio-r8a7742",
+				"renesas,rcar-gen2-gpio";
 		reg = <0 0xe6053000 0 0x50>;
 		interrupts = <0 7 IRQ_TYPE_LEVEL_HIGH>;
 		#gpio-cells = <2>;
@@ -336,7 +340,8 @@
 	};
 
 	gpio4: gpio@e6054000 {
-		compatible = "renesas,gpio-r8a7742", "renesas,gpio-rcar";
+		compatible = "renesas,gpio-r8a7742",
+				"renesas,rcar-gen2-gpio";
 		reg = <0 0xe6054000 0 0x50>;
 		interrupts = <0 8 IRQ_TYPE_LEVEL_HIGH>;
 		#gpio-cells = <2>;
@@ -349,7 +354,8 @@
 	};
 
 	gpio5: gpio@e6055000 {
-		compatible = "renesas,gpio-r8a7742", "renesas,gpio-rcar";
+		compatible = "renesas,gpio-r8a7742",
+				"renesas,rcar-gen2-gpio";
 		reg = <0 0xe6055000 0 0x50>;
 		interrupts = <0 9 IRQ_TYPE_LEVEL_HIGH>;
 		#gpio-cells = <2>;
-- 
2.7.4

